test/Elastic.Apm.Tests/ApiTests/ConvenientApiTransactionTests.cs

```diff
@@ -524,9 +526,7 @@ public void TransactionWithRequestDetailed()
 							new Request("GET",
 								new Url { Full = "https://elastic.co", Raw = "https://elastic.co", HostName = "elastic", Protocol = "HTTP" })
 							{
-								HttpVersion = "2.0",
-								Socket = new Socket { Encrypted = true, RemoteAddress = "127.0.0.1" },
-								Body = "123"
+								HttpVersion = "2.0", Socket = new Socket { Encrypted = true, RemoteAddress = "127.0.0.1" }, Body = "123"
```

Comment:
formatting?

Reply:
Yeah, this was Rider, no real change here - I don't have an opinion on formatting - my only opinion is that it should be standardized and never be an issue :)  I wonder why this happened on my machine and how we could standardize this. Maybe I'll check if `.editorconfig` could help here. 

I changed this back to the original.